pulser-core/pulser/sequence/sequence.py

```diff
@@ -326,6 +326,16 @@ def available_channels(self) -> dict[str, Channel]:
                 )
             }
 
+    @property
+    def is_empty(self) -> bool:
+        """States whether the sequence is empty.
+
+        Note:
+            This is equivalent to testing if the sequence duration is equal to
+            0.
+        """
+        return self.get_duration() == 0
```

Comment:
We have a bunch of `is_...` methods that are not properties, so maybe it would be best if this wasn't a property either